handler.py

```diff
@@ -1,50 +1,60 @@
 from conversions.conversions import Conversions
 from calculations.calculations import Calculations
-from plots import make_plot
+from plots.make_plot import make_plot
+import traceback
 
-conversions = Conversions("data\output2_linpot_2023-10-14_13-23-28.csv")
-calculations = Calculations("data\output2_linpot_2023-10-14_13-23-28.csv")
-plots = make_plot.make_plot()
+conversions = Conversions("data\\output2_linpot_2023-10-14_13-23-28.csv")
+calculations = Calculations("data\\output2_linpot_2023-10-14_13-23-28.csv")
 
 
 class handler:
 
-    def wheel_load_handler():
+    def wheel_load_handler(self):
+        # loads and prepares initial data
         conversions.convert_voltage_to_in()
         conversions.convert_time(conversions.data)
         conversions.clean_data()
         calculations.data = conversions.data
 
+        # calculating constants
         calculations.calculate_wheel_rate_front()
         calculations.calculate_wheel_rate_rear()
         calculations.calculate_wheel_rate_roll_front()
         calculations.calculate_wheel_rate_roll_rear()
         calculations.calculate_K_H_K_P()
-        calculations.K_H
         calculations.calculate_K_R_K_W()
-        calculations.K_W = calculations.K_R
         calculations.calculate_a_b()
-        calculations.wheel_load_const_a
         calculations.calculate_q_w()
-        calculations.w = calculations.q
-
-        calculations.calculate_force_front_left()
-        calculations.calculate_force_front_right()
-        calculations.calculate_force_rear_left()
-        calculations.calculate_force_rear_right()
-
-        plots.plot_wheel_load_vs_time(calculations)
-
-    def wheel_damper_handler():
+        try:
```

Comment:
this is the best place to put them, it encompasses all of the methods and calls, as a result if there is an issue then it will be shown